app/kubemci/pkg/gcp/loadbalancer/loadbalancersyncer.go

```diff
@@ -173,12 +173,18 @@ func (l *LoadBalancerSyncer) CreateLoadBalancer(ing *v1beta1.Ingress, forceUpdat
 			err = multierror.Append(err, cErr)
 		}
 
-		_, tpErr := l.tps.EnsureHttpsTargetProxy(l.lbName, umLink, certLink, forceUpdate)
+		tpLink, tpErr := l.tps.EnsureHttpsTargetProxy(l.lbName, umLink, certLink, forceUpdate)
 		if tpErr != nil {
 			// Aggregate errors and return all at the end.
 			fmt.Println("Error ensuring HTTPS target proxy:", tpErr)
 			err = multierror.Append(err, tpErr)
 		}
+		frErr := l.frs.EnsureHttpsForwardingRule(l.lbName, ipAddr, tpLink, clusters, forceUpdate)
+		if frErr != nil {
+			// Aggregate errors and return all at the end.
+			fmt.Println("Error ensuring https forwarding rule:", frErr)
```

Comment:
nit: i guess let's keep trying to print the same error message as we return: err = Errorf(blah), Print(err), err = Append(...)

Reply:
Done. Also updated existing error messages